src/js/graph/graph.js

```diff
@@ -12,7 +17,11 @@ function Graph(id) {
     this._sequenceNodeTypes = { };
     this._spanContainerNodeTypes = { };
     this._nodeTypes = { };
-    this._graphImpl = new GraphImpl({ multigraph: true });
+    this._graphImpl = new GraphImpl({
+        directed: true,
+        multigraph: true,
```

Comment:
Was experimenting with compound for parent node relationships, but graphlib limits you to 1 parent node, and since a token may have multiple parents I switched back to the child edge paradigm. Removed the related settings and constants.

Multigraph we use. Each node can now have multiple edges to another node, but only 1 of each type. This allows us to represent a token as both the "first" node and a "child" node of a SB node in the graphlib structure. The caveat is that we now have edges in the graph that aren't actually part of the pagi graph. If you look at the getEdges() function, https://github.com/digitalreasoning/pagijs/blob/feature/PAGI-170-refactor-edge-impl/src/js/graph/node.js#L82 , we're filtering out the child edges since they're an internal control edge. This is not great, but I think the benefits of having graphlib clean up the edge relationships  automatically when a node get's deleted is worth the cost.